shared/lib_battery_dispatch.cpp

```diff
@@ -61,7 +61,7 @@ dispatch_t::dispatch_t(battery_t * Battery, double dt_hour, double SOC_min, doub
 
 void dispatch_t::init(battery_t * Battery, double dt_hour, int current_choice, double t_min, int mode)
 {
-	_dt_hour = dt_hour; 
+	_dt_hour = dt_hour;
 	_current_choice = current_choice;
 	_t_min = t_min;
```

Comment:
Just checking that all battery fixes have been merged into patch and not develop - correct?